tests/sklearn/test_sklearn_models.py

```diff
@@ -464,6 +477,7 @@ def check_serialization_dumps_loads(model, x, use_dump_method):
         serialized_model_dict["serialized_value"].pop(attribute, None)
         re_serialized_model_dict["serialized_value"].pop(attribute, None)
 
+    # Check if the graphs are similar
     assert serialized_model_dict == re_serialized_model_dict
 
```

Comment:
Here you should check that `loaded_model.fhe_circuit.graph == model.fhe_circuit.graph`.

Reply:
yes fhe_circuit are not serialized

Reply:
I see you re-compile now I think that's what you needed to do. Thanks!